plugins/analytics-module-ga4/README.md

```diff
@@ -9,7 +9,11 @@ This plugin contains no other functionality.
 ## Installation
 
 1. Install the plugin package in your Backstage app:
-   `cd packages/app && yarn add @backstage/plugin-analytics-module-ga4`
+
+```sh
```

Comment:
While we're here, might as well add the `# From the Backstage root directory` line which has proven helpful for people